README.md

```diff
@@ -127,6 +132,8 @@ struct PurchaseProduct: SideEffect {
 }
 ```
 
+Note that the step `0` can be incapsulated in a protocol, should it be a prominent use case for the library/app.
```

Comment:
```suggestion
Note that the step `0` can be encapsulated in a protocol, should it be a prominent use case for the library/app.
```

Reply:
There is something not convincing in the `should it` form. This is not a question... 🤔 
what about:
 ```suggestion
Note that the step `0` can be encapsulated in a protocol, and that should be a prominent use case for the library/app.
```

Reply:
I think the `should it be` use is correct, what you proposed has a different meaning form what I intended.

I wanted to say that if using RSE is a prominent use case, then you can use a protocol. (cfr https://context.reverso.net/traduzione/inglese-italiano/Should+it+be)